app/views/components/table_rulings.scala.html

```diff
@@ -58,7 +58,7 @@
                 <tbody id="rulings-list-body">
                     @for((c: Case, index: Int) <- cases.results.zipWithIndex) {
                         <tr id="rulings-list-row-@{index}">
-                            <td id="rulings-list-row-@{index}-reference">@c.reference</td>
+                            <td id="rulings-list-row-@{index}-reference">GB@c.reference</td>
```

Comment:
I thought we stop prefixing the reference with "GB" a while back.    I presume that re-introduction is deliberate?

Reply:
After post-amigos this has been removed now, so no longer a thing